cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -522,6 +522,25 @@ def deploy_chatmail(config_path: Path) -> None:
         packages="postfix",
     )
 
+    # Add our OBS repository for dovecot_no_delay
+    files.put(
+        name = "Add Deltachat OBS GPG key to apt keyring",
+        src = importlib.resources.files(__package__).joinpath("obs-home-deltachat.gpg"),
+        dest = "/etc/apt/keyrings/obs-home-deltachat.gpg",
+        user="root",
+        group="root",
+        mode="644",
+    )
+
+    files.line(
+        name = "Add DeltaChat OBS home repository to sources.list",
+        path = "/etc/apt/sources.list",
+        line = "deb [signed-by=/etc/apt/keyrings/obs-home-deltachat.gpg] https://download.opensuse.org/repositories/home:/deltachat/Debian_12/ ./",
+        ensure_newline = True,
+    )
+
+    apt.update(name="apt update")
```

Comment:
We do an `apt update` in line 486 anyway, maybe leave this out and just move the two other operations above that?

Reply:
done